### DISCUSSION TITLE
Dropbox: Completely ignore folder as with "Selective Sync"?

### DISCUSSION THREAD

**Discussion by nobias** [iterate-ch/cyberduck#16277] [Q&A] [answered]
Hi! I'm using MountainDuck with Dropbox on Windows, and I have a few extremely large folders (300TB, tens of millions of files) in my account that I don't need at all, it's just old archived data.

Amazingly, MountainDuck seems to be dealing quite well with these massive folders, and I'm aware that by default, they won't take up any space on my local drive. However, MountainDuck does use quite a lot of RAM, 750 MB, probably to hold all the index and metadata.

So I would prefer to completely disable these folders, much like what the "Selective Sync" feature (not to be confused with the "Smart Sync" feature) in the native Dropbox client does: If a folder is disabled in Selective Sync, it won't ever show up in your synced folders (not even as an online-only folder) and is treated as if it wouldn't exist. I would assume that that would save quite a lot of RAM usage, since the number of files in those unused folders is just so much greater than the files I actually want to use.

So, is there any way for me to completely disable some folders? I'd be happy to tweak config files / issue commands / etc. Any hints greatly appreciated!

**Comment by dkocher** [comment, marked as answer]
The ~750MB memory usage is expected under regular usage regardless of the number of files to index. You can currently only exclude folders from sync using a [hidden configuration option](https://docs.cyberduck.io/mountainduck/preferences/#hidden-configuration-options) which is a regular expression to match filenames. The default on Windows is

```
fs.filenames.filter.name.regexp=[\\.]{1,2} .*[<>":\\?\\*\\|\\/\\\\].* \\.file-segments
```

You can add additional expressions separated by a single whitespace such as

```
fs.filenames.filter.name.regexp=[\\.]{1,2} .*[<>":\\?\\*\\|\\/\\\\].* \\.file-segments myarchive
```

**Comment by nobias** [comment]
Thank you very much, this is exactly what I was looking for.
